checks/contributors.go

```diff
@@ -38,6 +43,28 @@ func Contributors(c checker.Checker) checker.CheckResult {
 			if err != nil {
 				return checker.RetryResult(err)
 			}
+			// TODO - Figure out the real API for getting user orgs
+			url := fmt.Sprintf("https://api.github.com/users/%s/orgs", contrib.GetLogin())
+			resp, e := c.HttpClient.Get(url)
+
+			if e != nil {
+				c.Logf("unable to get org members for %s", contrib.GetLogin())
+			} else {
+				var orgs orgnaizations
+
+				err = json.NewDecoder(resp.Body).Decode(&orgs)
+
+				if err != nil {
+					c.Logf("unable to get  decode json for for %s org", contrib.GetLogin())
+				} else {
+					if len(orgs) > 0 {
+						companies[orgs[0].Login] = struct{}{}
```

Comment:
So, this does not do the filtering we do in the lines below [e.g. strip inc., llc, etc]. Probably need to move filtering to a helper function and then call in these 2 places.

Reply:
Took care of it. Thanks